chsdi/static/doc/source/services/sdiservices.rst

```diff
@@ -871,14 +875,28 @@ These projections are:
 Note:
 
 * Partly due to a limitation of the WTMS 1.0.0 recommendations, each *projection* has its own *GetCapabilities* document.
-* You have to use the hosts `wmts{10-14}.geo.admin.ch`.
+* You have to use the hosts `wmts{10-14}.geo.admin.ch`. This is done to avoid parsind every requests to determine which are
+  using native tiles and which are using reporjected tiles.
 * The same access restrictions apply as above.
 * The same `timestamps` are available in all projection. New `timestamp` are added to the former ones.
+* Some older WMTS client do not support `EPSG:4258 <http://www.epsg-registry.org/export.htm?wkt=urn:ogc:def:crs:EPSG::4258>`_ 
+  and use mistakenly *EPSG:4852*.
+* Reprojected tiles are generated *on-the-fly* with `MapProxy <http://mapproxy.org>`_. If you plan to heavily use this service, please
+  inform us in advance.
+* *MapProxy* uses the `Proj.4 <http://trac.osgeo.org/proj/>`_ library internaly to transform between datum, except for the reframe from  
+  **LV03/MN03** tiles which is *NTv2* grid based (`CHENyx06 <http://www.swisstopo.admin.ch/internet/swisstopo/en/home/products/software/products/chenyx06.html>`_)
+* Source for these reprojected tiles are the *native* **LV03/MN03** one. The only exception beeing *ch.kantone.cadastralwebmap-farbe* using a WMS service as its source.
+* Note all layer are available at all scales. You have to check for which **tileMatrixSets** a particuliar layer is defined. Your WMTS client may either stretch the
```

Comment:
`is defined, otherwise your WMTS client may either stretch the
instead of
`is defined. Your WMTS client may either stretch the`